jest/setup.js

```diff
@@ -73,6 +86,55 @@ jest.mock('@react-navigation/material-top-tabs', () => ({
   createMaterialTopTabNavigator: () => jest.fn(),
 }));
 
+const selectAccount = {
+  encoded: 'MFMCAQEwBQYDK2VwBCIEIKEjAyEAnHhgkOJZiuiE/50fAdahqbrxOp5h9zYzqJKPTYC/ff4=',
+  encoding: {content: ['pkcs8', 'sr25519'], type: ['none'], version: '3'},
+  address: 'G7UkJAutjbQyZGRiP8z5bBSBPBJ66JbTKAkFDq3cANwENyX',
+  meta: {
+    name: 'Test account name',
+    network: 'kusama',
+    isFavorite: false,
+    isExternal: true,
+    whenCreated: 1657023528984,
+  },
+};
+
+jest.mock('../src/polkadotApi/useAppAccounts', () => ({
+  useAppAccounts: () => ({
+    accounts: {},
+    networkAccounts: [selectAccount],
+    setAccounts: () => jest.fn(),
+  }),
+}));
+
+jest.mock('@atoms/activeAccount', () => ({
```

Comment:
I don't think we should mock this, recoil is a javascript library so we can use the real thing.. We can mock the `persistAtom` effect but for other things we should try to make our tests as much real as possible

Reply:
Here how we can test recoil related code: https://recoiljs.org/docs/guides/testing